### PR TITLE
New Minecraft Installer + Optifine

### PR DESCRIPTION
**Pull request by kylerchin**

### CODE REVIEW

**Comment by kylerchin** [comment]
This was created in response to issue #527

**Comment by kolgza** [comment]
I was the one who wrote the instructions on how to install the Flatpak version of Minecraft. I believe that we need to discuss whether we want new users to install Minecraft this way. At least, we need to have this discussion before the instructions to install the Flatpak version are outright removed.

**Comment by kylerchin** [comment]
What's the difference between this way and the flatpack way?

**Comment by kylerchin** [comment]
I'm not sure about the stability of the flatpak way and imo the Mojang supported way should be preferred since it's more supported by the developers. Open to input about it :)

**Comment by kolgza** [comment]
Well, my rationale for favoring the Flatpak version is that it's just easier for new users, and it's better to instruct new users to use the easier method.

I think we need some more outside input on this topic.

**Comment by jaxsnjohnson** [comment]
Hi, @kylerchin

I would love to work with you on expanding this article!  I was going to be writing an updated article with the wisdom I have learned hosting servers over the last 6 years.  I would love to work with you on it and I love the changes you have made.  LMK if you are interested in collaborating. 

_JJ - Sys76 Happiness Engineer_

**Comment by kylerchin** [comment]
> Hi, @kylerchin
> 
> I would love to work with you on expanding this article!  I was going to be writing an updated article with the wisdom I have learned hosting servers over the last 6 years.  I would love to work with you on it and I love the changes you have made.  LMK if you are interested in collaborating. 
> 
> _JJ - Sys76 Happiness Engineer_

Hi JJ! I would love to help! How do we start?

**Comment by jaxsnjohnson** [comment]
If you use Discord you are welcome to reach out to me here: `jaxsnjohnson#5416`

**Review by jaxsnjohnson — APPROVED** [review]
This looks good to me.

File: _articles/install-minecraft.md [hunk 1]
```diff
@@ -12,49 +12,62 @@ section: community
...
 
-You will then download the launcher. Go to the [minecraft.net](https://minecraft.net/en-us/download/) page to download or...
+You can download Optifine by going to the Optifine download page at https://optifine.net/downloads and clicking on "Download" for either the latest stable or preview version. 
 
+Then, you'll need to run the Java file. If you downloaded it into your Downloads folder, you can run it like 
 ```bash
-wget http://s3.amazonaws.com/Minecraft.Download/launcher/Minecraft.jar
+java -jar Downloads/Oprifine_version_number.jar
 ```
+If you downloaded a prevew version, the file name will start with `preview_`, so run `java -jar Downloads/preview_Optifine_version_number.jar` instead.
```
Comment (storyaddict) [inline _articles/install-minecraft.md:57]: typo: preview

Comment (storyaddict) [reply _articles/install-minecraft.md:57]:
```bash
java -jar Downloads/preview_Optifine_version_number.jar
```

**Review by storyaddict — CHANGES_REQUESTED** [review]
Couple of small formatting requests.

**Comment by storyaddict** [comment]
@kylerchin please rebase and make my requested changes, then we can merge! :)

**Comment by garnerlogan65** [comment]
Minecraft and other gaming articles should be more distinct from informational articles. Perhaps we could edit the site to promote social initiatives under the community section.

**Comment by gangwerz** [comment]
Closing for now. The conversation seems to have stalled.

**Pull request closed without merging**